### ISSUE TITLE
Student Reports: \cleardoublepage really needed?

### ISSUE THREAD

**Issue by philipoe** [ethz-asl/asl-student-templates#24]
The report template uses \cleardoublepage to align new chapters such that aligns the chapter after them to be a right-hand page (see https://github.com/ethz-asl/asl-student-templates/blob/master/Report/report.tex#L74). This seems to be mostly used when printing books (!), see http://libguides.lib.msu.edu/c.php?g=97090&p=908734 . However, in normal PDF-view or one-sided printing this can produce very strange empty pages.

@pfankhauser and others: Do we really need this in the template given that these theses will rarely be printed in book-style?

**Comment by pfankhauser** [comment]
Back in the days, we would print the reports, so the double clear made sense. I agree, that for PDF version only it's not necessary, but I wouldn't remove it completely. Maybe, we can add a global option to declare if you want them or not and can switch accordingly?

**Comment by philipoe** [comment]
We are talking about double-sided (front and back) printing, right (such that cleardoublepage would align the first page of a chapter to always be on the "front" side) ? I think a global option is fine. Default would then be no cleardoublepage?

**Comment by pfankhauser** [comment]
Exactly. Sure, we can set the default to no cleardoublepage and write as comment to use the other version of students print their reports.
